.github/workflows/ci.yml

```diff
@@ -121,8 +117,54 @@ jobs:
       with:
         arch: x86_64
         api-level: ${{ matrix.android-api }}
-        target: ${{ matrix.android-image-type }}
-        script: ./gradlew :androidTest:connectedCheck --no-daemon --no-build-cache
+        # Workaround for https://issuetracker.google.com/issues/267458959
```

Comment:
It looks massive, is it worth the complexity? (vs building on `26` only)?

Reply:
btw, most of the complexity is to get the logcat displayed and saved